src/core/Akka.API.Tests/CoreAPISpec.ApproveCluster.approved.txt

```diff
@@ -73,6 +73,7 @@ namespace Akka.Cluster
             public CurrentClusterState() { }
             public CurrentClusterState(System.Collections.Immutable.ImmutableSortedSet<Akka.Cluster.Member> members, System.Collections.Immutable.ImmutableHashSet<Akka.Cluster.Member> unreachable, System.Collections.Immutable.ImmutableHashSet<Akka.Actor.Address> seenBy, Akka.Actor.Address leader, System.Collections.Immutable.ImmutableDictionary<string, Akka.Actor.Address> roleLeaderMap) { }
             public System.Collections.Immutable.ImmutableHashSet<string> AllRoles { get; }
+            public bool HasMoreThanOneAppVersion { get; }
```

Comment:
Ran into this issue the other day with a large user cluster - looks like a good addition